mods/ra/rules/structures.yaml

```diff
@@ -1116,6 +1116,7 @@ HPAD:
 		ZOffset: 256
 		UpgradeTypes: primary
 		UpgradeMinEnabledLevel: 1
+	WithRearmAnimation:
```

Comment:
I thought so too, but it seems gahpad and nahpad don't have an `active` sequence.